src/Model/Filter/ILike.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Search\Model\Filter;
+
+use Cake\ORM\Query;
+
+class ILike extends Base
```

Comment:
This class is a copy of `Like` with just a single line change. So instead just add a config to `Like` to allow using either `LIKE` or `ILIKE`.  Then `Manager::ilike()` call just use 'Search.Like' with appropriate config.